### ISSUE TITLE
Browser Support

### ISSUE THREAD

**Issue by MeiYuanChen** [marko-js/marko#785]
Mako supports IE8+?
What about other browsers?
Not mentioned in README.md

**Comment by Joeao** [comment]
Some info in #438

**Comment by MeiYuanChen** [comment]
In order to support IE8-11,What I need to do?

**Comment by ulivz** [comment]
Maybe `marko` need to place the supported detail to the head section. I think.

**Comment by patrick-steele-idem** [comment]
Marko supports IE9+. Supporting IE8 is not possible because we are using some JavaScript language features (e.g. getters and setters) that are not available in IE8. We do need to make this clear in the documentation so I am keeping this issue opened.

**Comment by co16353sidak** [comment]
@patrick-steele-idem @ulivz would love to work on this issue if it isn't already taken 🥇

**Comment by DylanPiercey** [comment]
TODO: We should update the docs/website to point to `https://github.com/eBay/browserslist-config/blob/master/index.js`

Although I think ideally we load that file, and build up a link to https://browserslist.dev/?q=Q2hyb21lID49IDEwMywgRmlyZWZveCA%2BPSA3OCwgU2FmYXJpID49IDEzLCBFZGdlID49IDEwOSwgT3BlcmEgPj0gOTUsIENocm9tZUFuZHJvaWQgPj0gMTExLCBpb3Nfc2FmID49IDE0 (the `q` is just base64 encoded).

TLDR generating the url is
```ts
import queries from "@ebay/browserslist-config"

const url = `https://browserslist.dev/?q=${btoa(queries.join(", "))}`;
```
